apps/web/src/components/Menu/config/config.ts

```diff
@@ -67,7 +67,7 @@ const config: (
         },
         {
           label: t('Liquidity'),
-          href: '/liquidity',
+          href: '/liquidity/positions',
```

Comment:
I think we can leave the old /liquidity link instead of replaced to liquidity-old cc @chefjackson